charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```
@@ -302,7 +302,7 @@ rules:
   resources:
   - secrets
   - configmaps
-  - serviceaccounts
+  - serviceaccounts # TODO: Remove this in a future release
```

```suggestion
  - serviceaccounts # TODO(dimityrmirchev): Remove this in a future release
```

We try to add a note on who added the TODO along with the description why it was added